content/libraries.md

```diff
@@ -174,6 +174,7 @@ Some great folks have written their own libraries to help interact with Datadog.
 {: #community-integration-nginx-lua}
 
   * Emit custom metrics directly from NGINX configurations using the [nginx_lua_datadog][72] module in your LUA scripts.
+  * [lua-resty-dogstatsd][79] written by [mediba inc][80].
 
```

Comment:
Thanks @yoshida-mediba for building this as well as for adding it to the docs. Can you add a short sentence describing what the module is used for similar to the bullet points above it?